universe/Fleet.cpp

```diff
@@ -1156,6 +1156,23 @@ void Fleet::CalculateRouteTo(int target_system_id) {
     }
 }
 
+bool Fleet::Blockaded() const {
+    /** returns true if fleet is in a system and any exit is blocked */
```

Comment:
explain what being blockaded means in this context? presumably is that it is prevented from using at least one starlane entry to leave to leave its current system?